src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala

```diff
@@ -17,34 +17,35 @@ object FileMetadataFields {
 
   val SHA256ServerSideChecksum = "SHA256ServerSideChecksum"
   case class FileMetadata(filePropertyName: String, value: String) extends FileMetadataBase
-  case class AddFileMetadataInput(filePropertyName: String, value: String) extends FileMetadataBase
+  case class UpdateFileMetadataInput(filePropertyName: String, value: String) extends FileMetadataBase
```

Comment:
Hmm, I was thinking this but I kept it like this:

1. to be consistent with the other files and
2. I was thinking since they (`FileMetadata ` and `UpdateFileMetadataInput`) are subject to change (as requirements change e.g. we need to return another/a different property) using only one would make it harder to change in the future

Reply:
Fair point.